src/gsy_e/models/strategy/pv.py

```diff
@@ -221,17 +222,32 @@ def event_tick(self):
         self._settlement_market_strategy.event_tick(self)
         self._future_market_strategy.event_tick(self)
 
-    def set_produced_energy_forecast_kWh_future_markets(self, reconfigure=True):
+    def set_produced_energy_forecast_kWh_in_state(self, reconfigure=True):
+        self.set_produced_energy_forecast_kWh_spot_market(reconfigure)
+        self.set_produced_energy_forecast_kWh_future_market(reconfigure)
```

Comment:
I propose to remove the unit from all three method names to be consistent with the loads' equivalents. Also, the unit does not matter when calling the method (there will not be e.g. a `set_produced_energy_forecast_Wh_spot_market ` in the future).
Thanks!